### PR TITLE
fix: make footer copyright year dynamic

### PR DESCRIPTION
**Pull request by lucianmurmurache**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Change the footer year based on current year.

### Related issue

- #726

### CODE REVIEW

File: frontend/components/footer/flex/FooterFlex.vue [hunk 1]
```diff
@@ -51,7 +51,9 @@
         {{ $t("components.footer._global.powered-by-netlify") }}
       </a>
       <div class="mt-2 text-light-text dark:text-dark-text">
-        {{ $t("components._global.copyright") }}
+        {{
+          $t("components._global.copyright", { year: new Date().getFullYear() })
```
Comment (andrewtavis) [inline frontend/components/footer/flex/FooterFlex.vue:55]: I really like this, @lucianmurmurache! 👏 Super cool! Let us know if you come across other places where we can pass some variables to the i18n strings so clean them up, but also keep in mind that word order might need to change in other languages :)

**Review by andrewtavis — APPROVED** [review]
Thanks for the change and the really intuitive usage of the i18n string, @lucianmurmurache!

File: frontend/components/footer/flex/FooterFlex.vue (hunk 1 above)
Comment (lucianmurmurache) [reply frontend/components/footer/flex/FooterFlex.vue:55]: @andrewtavis sure I can take a look and create a PR if I find anything. In terms of the word order, if you meant for the change I made here, that should not be an issue, when writing the translation, all we need to do is to ensure that `{ year }` gets placed in the right spot and it should work like a charm!

Comment (andrewtavis) [reply frontend/components/footer/flex/FooterFlex.vue:55]: All sounds great, @lucianmurmurache!

**Pull request merged**